index.html

```diff
@@ -6,17 +6,115 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>My Blog</title>
     <link
       href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
       rel="stylesheet"
       type="text/css"
     />
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <title>Virgo Fashion Magazine</title>
   </head>
-  <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+  <body>
+    <header>
+
+      <section class="top-container">
+        <p class="logo">VIRGO</p>
+        <section class="top-right">
+        <p class="top-right-p1">BRITAIN</p>
+        <p class="top-right-p2">SUBSCRIBE</p>
+        <img class="hburger-img" alt="Picture of a hamburger menu" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQhhwlrve66B8GHVhBRth6kDr6ml2SQR73R5g&usqp=CAU">
```

Comment:
Alt text is difficult you generally dont want to describe the image instead state its function. Here something like "Menu" or "Display main menu" would be more suitable